src/main/resources/init_db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE `taxi_db`;
```

Comment:
I would recommend adding 'IF NOT EXISTS' to the SQL queries. It will save us from unexpected behavior